### PR TITLE
Troubleshoot deploying with Travis CI

### PR DESCRIPTION
**Pull request by hillstew**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,5 +11,6 @@ script:
...
-    app: zeroandbeyond
+    secure: lUIuDVxGzkgAYOzdzbJlHdFr8utUn5lA2A4/a1XNCL5gpwNsvLQfKg3Q6DjEj9Lzez6AMGSn9MRrd8JAAanatkYtEwer6kzv3yJgGgcVbU6RIm99YsJ7rPGzSoFPt7f6KdyIUlrCPtqxZscsTzTQOGhNL3CPB/JaB9SSoeXj9pEhacM4OJv0mioL7/dYrPNGURbFsbl4P5byH8WtYYGybh+7Rr7yLlMqE6vFBCos4mwICdw3WLHkGy8Q/oH1wZtXRLr4ZqB851o7roSd9/DPg4k7E/FG0b2CLzD2iPETkOyxf7T2y9KZcTkmrJLwT5WuAWm4rb88v/e/1aU/lWUEaz+UnxCSjDcqPW4L4KI/I/YdQVkDEAAC2e1eqqVKR5VBVxMw26xOfhznsLwg1EqW9qpv3FnhAp9poV30u7rcrv0T+31TOLxCpNSQbPF39ZXOtNv7h2BFGWKqcdWzcGj6+RJQBySQ/0DlIEaJnY41SWCFRWIG5V8PP8LV29ugcJNYnkHJ5STtFtldo6Ie5yFn8jyNRVBaVkiMt5CyrNtGnfLdAU/t3mNMiA6B9GVPZrukFWbE7XaQkd37BXZq48tcavMrlBdDt/JvGHCA9x1y9KBCfQnRq6dYa1HGEGp5NkjNSIu62vI27rqHXhyXnKiLWrFh4LKYD3lVJseDAJBneO0=
```
Comment (lrs8810) [inline .travis.yml]: @hillstew Do you need to add `app:  zeroandbeyond` and  possibly `deploy: skip_cleanup: true`?

See this Travis CI discussion: https://travis-ci.community/t/no-stash-entries-found-and-api-request-failed/119

Comment (hillstew) [reply .travis.yml]: I had it and it failed 😞 but maybe the `skip_cleanup`. I will make it as a suggestion and then would you approve it?

Comment (lrs8810) [reply .travis.yml]: Approved!

Comment (lrs8810) [reply .travis.yml]: Approved, I will merge the PR.

**Pull request merged**